js/client-chat.js

```diff
@@ -657,6 +669,11 @@
 					}
 					switch (targets[0]) {
 					case 'add':
+					case 'roomadd':
+						var thisroom = (targets[0] === 'add' ? 'global' : (this.type === 'battle' ? 'battle' : this.id));
+						var hlkey = (thisroom === 'global' ? 'global' : Config.server.id + '-' + Config.server.port + ':' + thisroom);
```

Comment:
You're going to generate things like `127.0.0.1:8001-8001:lobby` and `gold-8000:lobby`.